scripts/insert_into_appointments.sql

```diff
@@ -1 +1,30 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(on_date, start_at, comment, patient_id, doctor_id)
+  SELECT DATE '2016-08-12', TIME '10:00', '', p.id, d.id
+  FROM patients AS p, doctors AS d
+  WHERE p.family_name = 'Wilcox'
+    AND p.given_name = 'Marsha'
+    AND d.family_name = 'Hilston'
```

Comment:
This row doesn't get inserted because of this typo.  `Hilston` should be `Hilton`.